trin-core/src/jsonrpc/service.rs

```diff
@@ -219,9 +221,13 @@ fn handle_request(
             "result": "trin 0.0.1-alpha",
         })
         .to_string()),
-        "test_historyNetwork" => dispatch_portal_request(obj, portal_tx),
-        "test_stateNetwork" => dispatch_portal_request(obj, portal_tx),
         _ if obj.method.as_str().starts_with("discv5") => dispatch_portal_request(obj, portal_tx),
```

Comment:
Well, now after all this effort to use enums and improve our parsing - I gotta say it's a bit weird to still have this semi-string parsing step. Do you see any obvious way we could consolidate them?

Reply:
Everything is now send to `dispatch_portal_request`, where the parsing is done. If it is `InfuraEndpointKind`, it is then send to `dispatch_infura_request` for processing. When we start to deal with more `eth_` endpoints, we may refactor this more.